radicle-httpd/src/lib.rs

```diff
@@ -79,3 +100,58 @@ pub async fn run(options: Options) -> anyhow::Result<()> {
         .await
         .map_err(anyhow::Error::from)
 }
+
+#[derive(Clone)]
+struct IdCounter(Arc<AtomicU64>);
```

Comment:
```suggestion
struct RequestId(Arc<AtomicU64>);
```
Better to say what it is, than how it works imo